src/ui/components/ActivitySelecter/ActivitySelecter.tsx

```diff
@@ -52,6 +52,9 @@ export enum FullScreenComponent {
     FreeInput,
 }
 
+const selectedIdNewActivity = "selectedIdNewActivity";
+const selectedValue = "selectionValue - label";
```

Comment:
Naming may be confusing  replace selectedValue by selectedValueLabel or selectedLabelNewActivity if it makes sense